test/server.test.js

```diff
@@ -229,6 +229,33 @@ test('rm route and clear cached route', function (t) {
     });
 });
 
+test('rm one version of the routes, other versions should still work',
```

Comment:
Can you prepend this test name with the issue it is fixing? 'GH-1171: rm one version...'